x-pack/plugins/observability/server/services/slo/sli_client.test.ts

```diff
@@ -417,3 +430,23 @@ describe('SLIClient', () => {
     });
   });
 });
+
+expect.extend({
```

Comment:
💬 Dealing with date in test is always annoying. This expect function compares the difference between two dates and allow for 2min differences.